Makefile

```diff
@@ -193,13 +194,13 @@ install-ngf-local-build-with-plus: build-images-with-plus load-images-with-plus
 
 .PHONY: helm-install-local
 helm-install-local: ## Helm install NGF on configured kind cluster with local images. To build, load, and install with helm run make install-ngf-local-build.
```

Comment:
I wonder if we use this helm targets at all. Could find any references (like docs or other Makefiles) by searching https://github.com/search?q=repo%3Anginxinc%2Fnginx-gateway-fabric+helm-install-local&type=code ?

Reply:
We use it with `make debug-install-local-build`, which is referenced in the debugging guide 😁